code/framework/src/scripting/engines/node/engine.cpp

```diff
@@ -217,7 +217,7 @@ namespace Framework::Scripting::Engines::Node {
         // Create the execution environment
         node::EnvironmentFlags::Flags flags = (node::EnvironmentFlags::Flags)(node::EnvironmentFlags::kOwnsProcessState);
         _gamemodeData                           = node::CreateIsolateData(_isolate, uv_default_loop(), GetPlatform());
-        std::vector<std::string> argv       = {"mafiahub-resource"};
+        std::vector<std::string> argv       = {"mafiahub-gamemode"};
```

Comment:
idk if it revealant here

Are we talking about a V8 resource or a server resource?